config/crds/workflows_crd.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Can we refer to the release version from the argo workflow repo?

Reply:
Done as suggested. Thanks.